more/free-programming-interactive-tutorials-en.md

```diff
@@ -30,6 +32,7 @@
 * [Language Agnostic](#language-agnostic)
     * [Operating Systems](#operating-systems)
 * [LaTeX](#latex)
+* [Linux](#linux)
```

Comment:
goes under operating systems.
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.